tasks/protobuf.py

```diff
@@ -145,9 +150,31 @@ def generate(ctx, do_mockgen=True):
             ctx.run(f"git apply {switches} --unsafe-paths --directory='{pbgo_dir}/{pkg}' {patch_file}")
 
     # Check the generated files were properly committed
-    updates = ctx.run("git status -suno").stdout.strip()
-    if updates:
+    git_status = ctx.run("git status -suno", hide=True).stdout
+    # git_status = ctx.run("git status -suno | grep -E 'pkg/proto/pbgo.*.pb(.gw)?.go'").stdout.strip()
+    proto_file = re.compile(r"pkg/proto/pbgo/.*\.pb(\.gw)?\.go$")
+    if any(proto_file.search(line) for line in git_status.split("\n")):
         raise Exit(
-            "Generated files were not properly committed. Please run `inv protobuf.generate` and commit the changes.",
+            f"Generated files were not properly committed.\n{git_status}\nPlease run `inv protobuf.generate` and commit the changes.",
             code=1,
         )
+
+
+def check_dependencies(ctx):
```

Comment:
All sub methods and messages mention tools. Should we name this function, `check_tools` ?